resources/views/filament/forms/components/campaigns/actions/proactive-alert.blade.php

```diff
@@ -52,7 +52,7 @@
         </div>
         <div class="flex flex-col pt-3">
             <dt class="mb-1 text-sm text-gray-500 dark:text-gray-400">Status</dt>
-            <dd class="text-sm font-semibold">{{ $action['status'] }}</dd>
+            <dd class="text-sm font-semibold">{{ $action['status_id'] }}</dd>
```

Comment:
The User isn't going to know what this ID is. So let's change this to something like a `findOrFail` on the status and display the name.